style.css

```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&display=swap');
+
+.toolbar {
+  position: absolute;
+  width: 375px;
+  height: 46px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
```

Comment:
- [OPTIONAL] Based on the requirements, this section should be aligned using a flexbox. Using flexbox makes your design responsive across various screen sizes. I would advise you to implement this suggestion. 👍